index.html

```diff
@@ -0,0 +1,118 @@
+                    <a href="/Capstone-Project/index.html"><img src="./assets/imgs/logo.png" alt="Logo"></a>
+                </div>
+                <div class="menu">
+                    <ul class="menu-list-ul">
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <!-- Main Content Section -->
+        <main>
+        <section class="hero-section">
+            <div class="hero-div">
+                <h1 class="thin-text">"Hello! Sharing Knowledge around the Globe"
+                    <br><h1 class="thick-text">Awareness Summit about web development 2023</h1>
+                </h1>
+                <span class="description-about-summit">
+                     A joyful and informative enviroment for sharing creative ideas about the advancements of technology.
+                </span>
+                <span class="summit-date">
+                    2023.10.15(FRI)
+                </span>
+                <span class="summit-location">
+                    @ABC Avenue NY
+                </span>
+            </div>
+        </section>
+                        <!-- main program section -->
+        <section 
+        class="main-program-section">
+            <div class="main-program-div">
+                 <h2>Main Program</h2>
+                <div class="main-program-item-container">
+                    <div class="main-program-item">
+                     
+                        <h3>Lecture</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                     
+                        <h3>CC Exhibition</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                        
+                        <h3>Forum</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                        
+                        <h3>Workshop</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                       
+                        <h3>CC Ignite</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                </div>
+            </div>
+        </section>
+                    <!-- feature speaker section -->
```

Comment:
- [ ] Please not that these are atricles and should be marked as such in the HTML code . instead of having them in divs please put the in article tags like 
```
<article>
 .. content
</article>
```

Reply:
Resolved